tf2onnx/onnx_opset/math.py

```diff
@@ -253,10 +253,16 @@ def version_1(cls, ctx, node, **kwargs):
         #     output = input / (bias + alpha * sqr_sum) ** beta
         depth_radius = node.get_attr("depth_radius")
         if depth_radius:
-            size = depth_radius.i
+            size = depth_radius.i * 2 + 1
         else:
-            size = 5
+            size = 5 * 2 + 1
+
         node.set_attr("size", size)
+        node.set_attr("alpha", size * node.get_attr("alpha").f)
+
+        ctx.insert_new_node_on_input(node, "Transpose", node.input[0], perm=[0, 3, 1, 2])
```

Comment:
instead of [0, 3, 1, 2], you could use constants defined in https://github.com/onnx/tensorflow-onnx/blob/7363ef6e0fa3c587c5337035c1bd26f07a9d34f4/tf2onnx/constants.py#L33